tests/gss/new_clients_k5.conf.in

```diff
@@ -0,0 +1,4 @@
+include @objdirabs@/krb5.conf
```

Comment:
Oh, does `include` require absolute paths?

Reply:
I recall it did, the man page also mentions absolute: https://github.com/heimdal/heimdal/blob/122b1720ada15ed92d229a898acfff7c6625c5bf/lib/base/config_file.c#L423